build/templates/run-pester-tests.yml

```diff
@@ -2,6 +2,12 @@ parameters:
   projectName: ''
 
 steps:
+  - pwsh: 'wget -O - https://aka.ms/install-powershell.sh | sudo bash'
+    displayName: 'Update Powershell on Linux'
+    condition: eq( variables['Agent.OS'], 'Linux' )
+  - powershell: 'Invoke-Expression -Command "& { $(Invoke-RestMethod -Uri ''https://aka.ms/install-powershell.ps1'') } -UseMSI -quiet" '
+    displayName: 'Update Powershell on Windows'
+    condition: eq( variables['Agent.OS'], 'Windows_NT' )
```

Comment:
We needed PowerShell Core, yes?
Great job in updating them (window/linux) both. 👍